lib/laa_crime_schemas/types/types.rb

```diff
@@ -118,6 +156,19 @@ module Types
     OFFENCE_CLASSES = %w[A K G B I J D C H F E].freeze
     OffenceClass = String.enum(*OFFENCE_CLASSES)
 
+    OwnershipType = String.enum(*%w[with_partner landlord other_person sole])
+
+    SavingsType = String.enum(*%w[
+                                cash
+                                other_person_account
+                                certificates
+                                shares
+                                trusts
+                                policies
+                              ])
+
+    EmploymentStatusType = String.enum(*%w[employed self-employed none])
```

Comment:
Worth adding in the others we're collecting here?
partnership
director
private_shareholder

Reply:
Perhaps in addition to the employment_status we have  an array "employment_types" to record all the employment types of the client. E.g. ``` "employment_types": ["director", "private_shareholder", "paye"]

Reply:
@arthurashman  if it is okay, I'd rather leave this (how we capture employment income sources) until the self-employed journey is finessed.

Reply:
Ok, all good. We don't need it just yet anyhow